mypy/dmypy_server.py

```diff
@@ -465,4 +558,17 @@ def get_meminfo() -> Dict[str, Any]:
         meminfo = process.memory_info()
         res['memory_rss_mib'] = meminfo.rss / MiB
         res['memory_vms_mib'] = meminfo.vms / MiB
+        if sys.platform == 'win32':
```

Comment:
@gvanrossum I wasn't entirely sure what to do here, so I refactored things to be a bit more reliant on psutil existing. I could manually get the max memory used with win32 APIs if psutil isn't installed if you prefer, but I felt this was cleaner.